docs/documentation/development/bindings/categories.md

```diff
@@ -14,7 +14,44 @@ The thing type definition allows to specify a category. User interfaces can pars
 
 | Category        | Description                                          |
 |-----------------|------------------------------------------------------|
-
+| Aircondition    | Air condition devices incl. Fans |
```

Comment:
Aircondition might be a bit specific, thinking that combined HVAC appliances are very wide-spread (especially in the US) - so HVAC might be a better choice.